tests/test_multiple_neutrals.py

```diff
@@ -0,0 +1,34 @@
+from carsons.carsons import CarsonsEquations
+from tests.helpers import LineModel
+
+
+def test_dual_neutral_model():
+
+    model = CarsonsEquations(LineModel({
+        #    resistance   gmr         (x, y)
+        #   ==========================================
+        "A":  (0.000115575, 0.00947938, (0.762, 8.5344)),
+        "B":  (0.000115575, 0.00947938, (2.1336, 8.5344)),
+        "C":  (0.000115575, 0.00947938, (0.0, 8.5344)),
+        "N1": (0.000367852, 0.00248107, (0.0, 7.3152)),
+        "N2": (0.000367852, 0.00248107, (1.2192, 7.3152)),
+    }))
+    z_primitive = model.build_z_primitive()
+    assert z_primitive.shape == (5, 5)
+
+    # need to get a benchmark case and test the accuracy of the calculation
```

Comment:
Will this be done in this PR too?

Reply:
I'm not sure. We need to find an example in IEEE or somewhere that we could use as a benchmark. The thing is, we already support this configuration, although it isn't tested.

Reply:
Ok I found what i think is a decent way to test this case --- I used the ABCN model but swapped the phase labelling to be A/B/N1/N2. This should give us the same z_primitive matrix, but with an extra column/row of zeros for C.